File: platform-101/01-test-claude.js
import "dotenv/config";
import Anthropic from "@anthropic-ai/sdk";

const client = new Anthropic();

const msg = await client.messages.create({
  model: "claude-haiku-4-5",
  max_tokens: 1024,
  messages: [{
    role: "user",
    content: "Hello, Claude",
  }],
});

console.log(msg);

File: platform-101-python/01_test_claude.py
from dotenv import load_dotenv
load_dotenv()

import anthropic

client = anthropic.Anthropic()

msg = client.messages.create(
    model="claude-haiku-4-5",
    max_tokens=1024,
    messages=[{"role": "user", "content": "Hello, Claude"}],
)

print(msg)
